server/resource_server/models/user.go

```diff
@@ -1,13 +1,15 @@
 package models
 
 type User struct {
-	ID        uint   `gorm:"primaryKey; unique; autoIncrement; not null" json:"id"`
-	Email     string `gorm:"column:email; unique; not null" json:"email"`
-	Username  string `gorm:"column:username; unique; not null" json:"username"`
-	Password  string `gorm:"column:password; not null" json:"password"`
-	FirstName string `gorm:"column:first_name; not null" json:"first_name"`
-	LastName  string `gorm:"column:last_name; not null" json:"last_name"`
-	Salt      string `gorm:"column:salt; not null" json:"salt"`
+	ID               uint   `gorm:"primaryKey; unique; autoIncrement; not null" json:"id"`
+	Email            string `gorm:"column:email; unique; not null" json:"email"`
```

Comment:
You said that we will no longer save email, is this coming in some next PRs or we should remove `email` column from user?

Reply:
Yes, this one is for the upcoming PR. I had to remove email from the registration form, which required additional changes, that's why I split it into two PRs.